### PR TITLE
Add missing spans section

### PR DESCRIPTION
**Pull request by MariadeAnton**
Add missing spans section
Add navTitles to collector and protocol pages while I'm here

## Affected Components
* [X] Docs

## Pre-Requisites
* [x] Code is linted (`npm run lint`)

## Notes for the Reviewer

Review the new page: https://checklyhq-com-git-mda-missing-spans-checkly.vercel.app/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans/ , thank you!

### CODE REVIEW

File: site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+This guide will walk you through the process of identifying missing spans, understanding their root causes, and implementing solutions to ensure your OpenTelemetry setup with Checkly provides complete and accurate tracing information. 
+
+## Missing spans
+
+Missing spans, also known as orphan spans, are segments of a trace that are disconnected from either their parent spans or from the root span.
+
+An instrumented distributed system, will emit spans creating a hierarchical chain that follows the flow of the request through the different components and services.
+
+When a span is missing, they create a whole in this chain, making it difficult to diagnose issue and understand the full picture.
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md:20]:
```suggestion
When a span is missing, they create a hole in this chain, making it difficult to diagnose issues and understand the full picture. If this happens, you will notice a span missing from the visualization in Checkly.
```

File: site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+beta: true
+---
+
+This guide will walk you through the process of identifying missing spans, understanding their root causes, and implementing solutions to ensure your OpenTelemetry setup with Checkly provides complete and accurate tracing information. 
+
+## Missing spans
+
+Missing spans, also known as orphan spans, are segments of a trace that are disconnected from either their parent spans or from the root span.
+
+An instrumented distributed system, will emit spans creating a hierarchical chain that follows the flow of the request through the different components and services.
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md:18]:
```suggestion
An instrumented distributed system will emit spans, creating a hierarchical chain that follows the flow of the request through the different components and services.
```

File: site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+When viewing trace details, a span might be missing in the Checkly visualization. 
+Using contextual span data, about the trace it belongs to, and it's parent span; Checkly displays any missing pieces on the span chain, so that you can identify which application or service might be incorrectly instrumented. 
+
+![Trace with missing spans](/docs/images/otel/traces-missing-spans.png)
+
+
+## Troubleshooting missing spans
+
+Missing spans can happen due to various reasons.  Wether you're sending data to Checkly directly from your applications and services, or through the OpenTelemetry Collector, these are steps to confirm everything is properly setup.
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md:30]:
```suggestion
Missing spans can happen due to various reasons. Whether you're sending data to Checkly directly from your applications and services, or through the OpenTelemetry Collector, these are steps to confirm everything is properly set up.
```

File: site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+  
+      For example, to allow HTTP traces to be sent out: 
+      ```bash
+      A OUTPUT -p tcp --dport 4318 -j ACCEPT
+      ```
+
+
+### Incomplete instrumentation
+   
+   To see the full picture, and identify the root cause of a problem faster; ensure [your applications and services are instrumented with the OpenTelemetry SDK](docs/traces-open-telemetry/instrumenting-code/).
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md:114]:
```suggestion
   To see the full picture, and identify the root cause of a problem faster, ensure [your applications and services are instrumented with the OpenTelemetry SDK](docs/traces-open-telemetry/instrumenting-code/).
```

File: site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+## Missing spans
+
+Missing spans, also known as orphan spans, are segments of a trace that are disconnected from either their parent spans or from the root span.
+
+An instrumented distributed system, will emit spans creating a hierarchical chain that follows the flow of the request through the different components and services.
+
+When a span is missing, they create a whole in this chain, making it difficult to diagnose issue and understand the full picture.
+
+When viewing trace details, a span might be missing in the Checkly visualization. 
+Using contextual span data, about the trace it belongs to, and it's parent span; Checkly displays any missing pieces on the span chain, so that you can identify which application or service might be incorrectly instrumented.
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md:23]:
```suggestion
Checkly uses contextual span data—such as the trace it belongs to and its parent span—to display any missing pieces in the span chain. This helps you identify which application or service may be incorrectly instrumented.
```

File: site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+
+Missing spans can happen due to various reasons.  Wether you're sending data to Checkly directly from your applications and services, or through the OpenTelemetry Collector, these are steps to confirm everything is properly setup.
+
+### Verify sampling strategy
+
+   Using Checkly Traces, we recommend sampling for the tracestate header `checkly=true`, which will reduce your Egress/Ingress costs and ensure that all the spans that were originated through a check are there.
+
+   #### Sampling in the OpenTelemetry Collector
+
+      a. Make sure that you're filtering out non-checkly spans:
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md:38]: Is this the desired formatting?
Screenshot 2024-10-24 at 08 17 41

File: site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+      service:
+         pipelines:
+            traces:
+               exporters: [otlp/checkly]
+      ```
+
+   #### Exporting directly from your Application's code
+      Confirm the environment variables use the right endpoint and authorization:
+
+      ```bash
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md:89]: The formatting didn't work properly here
Screenshot 2024-10-24 at 08 19 30

File: site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md (hunk 1 above)
Comment (sujaya-sys) [reply site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md:20]: Note: This is just a suggestion from my side, I feel like that'll help connect the dots, feel free to ignore if it doesn't make sense to you :)

File: site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md (hunk 5 above)
Comment (sujaya-sys) [reply site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md:23]: I tripped over this sentence initially - again just a suggestion to restructure it for clarity. Feel free to ignore :)

File: site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md (hunk 1 above)
Comment (MariadeAnton) [reply site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md:20]: This is great, also thank you for catching the typo!

File: site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md (hunk 7 above)
Comment (MariadeAnton) [reply site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md]: Aha, indentation, thanks!

File: site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+
+## Missing spans
+
+Missing spans, also known as orphan spans, are segments of a trace that are disconnected from either their parent spans or from the root span.
+
+An instrumented distributed system will emit spans, creating a hierarchical chain that follows the flow of the request through the different components and services.
+
+When a span is missing, they create a hole in this chain, making it difficult to diagnose issues and understand the full picture. If this happens, you will notice a span missing from the visualization in Checkly.
+
+When viewing trace details, a span might be missing in the Checkly visualization.
```
Comment (sujaya-sys) [inline site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md:22]: Duplicated sentence - we're saying the same in the section above

Comment (MariadeAnton) [reply site/content/docs/traces-open-telemetry/importing-traces/troubleshooting-missing-spans.md:22]: Oh, I missed this when adding your suggestion, didn't realize this was the case. 😅  Thank you!
```suggestion

```

**Review by sujaya-sys — APPROVED** [review]
LGTM!

**Pull request merged**